frontend/src/services/api/timeouts.ts

```
@@ -0,0 +1,93 @@
+
+// Retry configuration
+export const RETRY_CONFIG = {
+  maxRetries: MAX_RETRIES,
+  initialDelay: RETRY_DELAY,
+  maxDelay: 10000, // 10 seconds max delay
+  retryableErrors: ['ECONNABORTED', 'ETIMEDOUT', 'ENOTFOUND', 'ENETUNREACH'],
+  retryableStatusCodes: [408, 429, 500, 502, 503, 504],
+};
+
+// Helper to get timeout for a specific endpoint
+export function getTimeoutForEndpoint(url: string, method: string = 'GET'): number {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)